fuel-reactor/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'kotlin'
```

Comment:
I think this needs to be modified a bit after the Kotlin-dsl made it into master #423, right?

Reply:
```
plugins {
   kotlin("jvm") 
}
```

Reply:
Changed in e1180d2d05b6ad2dc8e2ed0c8fcc6165fc57a7f2